tests/integration/datasetio/test_datasetio.py

```diff
@@ -99,3 +99,4 @@ def test_get_rows_paginated(llama_stack_client):
     assert isinstance(response.rows, list)
     assert len(response.rows) == 2
     assert response.next_page_token == "5"
+    llama_stack_client.datasets.unregister("test_dataset")
```

Comment:
should this be registered as test case cleanup (in case the test fails and doesn't reach here)?

Reply:
I think this is being done with fixtures with pytest: https://docs.pytest.org/en/7.1.x/how-to/fixtures.html

Reply:
done